modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1564,6 +1566,72 @@ public void testSearchObjectEntries() throws Exception {
 		Assert.assertEquals(0, baseModelSearchResult.getLength());
 	}
 
+	@Test
+	public void testSystemObjectExtendedProperties() throws Exception {
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), User.class.getName());
+
+		_objectFieldLocalService.addCustomObjectField(
```

Comment:
Don't we have to delete these custom fields after?

Reply:
I tested and these fields are automatically deleted after running this test

Reply:
Great. Thanks. 🙏🏻